webcompat/templates/dashboard/triage/filters.html

```diff
@@ -0,0 +1,67 @@
+<style>
+  .wc-Filters {
+    font-size: 16px;
+    font-size: 1rem;
+    margin: 1.2em 0;
+  }
+
+  .wc-Filters-form {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: wrap;
+        flex-wrap: wrap;
+  }
+
+  .wc-Filter {
+    padding: 1em;
+  }
+</style>
+
+<section class="wc-Filters" id="js-Filters">
+  <form
+    class="wc-Filters-form"
+    id="js-Filters-form"
+    method="get"
+    action=""
+    role="form"
+  >
+    <div class="wc-Filter">
+      <select
+        class="wc-Filter-select"
+        name="browsers"
+        id="js-Filter-browsers"
+      >
+        <option value="">All browsers</option>
+        <option value="firefox">Firefox</option>
+        <option value="chrome">Chrome</option>
```

Comment:
we have a list somewhere on the python side to hydrate this list maybe ?

Reply:
We don't but I can try to make sure that the list is limited to a certain number of things. I need to check something.

so currently in the tests I do:

https://github.com/webcompat/webcompat.com/blob/dashboard/tests/test_dashboard.py#L80-L83

But I don't test for `browser-firefox-mobile`, it will return `firefox-mobile` for now, but I can make it return `firefox` only.

What do you think?

@miketaylr @adamopenweb @MDTsai for the main triagers.

Reply:
yes, sounds good.  👍 

For now I add edge and safari.